modules/virtual_network_gateway/variables.tf

```diff
@@ -200,41 +229,57 @@ variable "local_bgp_settings" {
       }
     }
   }
-
+  ```
   EOF
-  type        = any
+  type = object({
```

Comment:
all the properties are optional but the variable is required?

Reply:
you are right, fixed